BossMod/Modules/Endwalker/Trials/T02Hydaelyn/Exodus.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+
+namespace BossMod.Endwalker.Trials.T02Hydaelyn
+{
+   class Exodus : BossComponent
+    {
+        private int crystalsdestroyed;
+        private bool casting;
```

Comment:
This is not a cast? This looks like something that should be implemented via state machine.

Reply:
Exodus is a cast, it says cast event in the raw ops
I don't understand how the statemachine works so i am trying to avoid using more than trivial phase, plus the statemachine doesn't add a blue hint or predicted damage

Reply:
Blue hint is redundant for raidwides covered by statemachines.

Predicted damage is done by checking next state with raidwide hint.

Reply:
noone is looking at the ugly wall of white text in the state machine though, maybe if it would be presented better like in cactbot it would be worth the effort learning it